web/styles/compose.css

```diff
@@ -805,6 +805,41 @@ a.compose_control_button.hide {
     margin-right: 4px;
 }
 
+#send_later {
+    float: right;
+    color: hsl(0deg 0% 100%);
+    border-radius: 0 4px 4px 0;
+    border-left: 1px solid hsl(213deg 14% 12% / 15%);
+    padding: 0;
+    margin: 0;
+
+    .fa {
+        padding: 4.5px 4px;
+
+        &::before {
+            position: relative;
+            top: -1px;
+        }
+    }
+
+    &:hover,
+    &:focus {
+        box-shadow: none;
+    }
+
+    /* We want to change background color on hover but do not
+       want to do so on focus since focus stays on the button
+       when user closes the popover and the compose looks weird
+       with one button with different color for no reason. */
```

Comment:
I think it's possible a better fix here would be to move focus to the compose box's text area when the popover closes; though that probably requires some care -- we only will want that if the focus wasn't set somewhere else by the click that closes the popover.

Reply:
Okay, added this to fix it:
```js
        onUntrigger() {
            // This is only called when the popover is closed by clicking on `target`.
            $("#compose-textarea").trigger("focus");
        },
```